src/lib/server-utilities.js

```diff
@@ -1,9 +1,40 @@
 import fs from 'fs';
+
+// md5Hash packages
 import crypto from 'crypto';
 
+// compressJPEG packages
+import imagemin from 'imagemin';
+import imageminJpegRecompress from 'imagemin-jpeg-recompress';
+
 export function md5Hash(file) {
   const hashInstance = crypto.createHash('md5');
   // readFileSync in the syncronous version of readFile
   const fileContents = fs.readFileSync(file, 'utf8');
   return hashInstance.update(fileContents).digest('hex');
 }
+
+
+/*
+ *  Compresses JPEG image passed as file path and overwrite uncompressed file
+ *  Returns imagemin Promise
+ */
+export function compressJPEG(filePath) {
+  const fileDirectory = filePath.substring(0, filePath.lastIndexOf('/'));
+  return imagemin([filePath], fileDirectory, {
+    plugins: [
+      imageminJpegRecompress(true), // Favor accuracy over speed
+    ],
+  });
+}
+
+/* Deletes file passed as file path */
+export function deleteFile(filePath) {
+  return new Promise(() => {
```

Comment:
The promise is not really worth anything if it never resolves, right? Probably add a resolve if no err on the `fs.unlink` callback